mono/metadata/class-internals.h

```diff
@@ -1286,7 +1286,7 @@ MONO_API void mono_class_describe_statics (MonoClass* klass);
 /* method debugging functions, for use inside gdb */
 MONO_API void mono_method_print_code (MonoMethod *method);
 
-char *mono_signature_full_name (MonoMethodSignature *sig);
+char *mono_signature_full_name (MonoMethodSignature *sig) MONO_PROFILER_API;
```

Comment:
Minor style nitpick: Shouldn't the `MONO_PROFILER_API` be applied before the return type? That's what we usually do for `MONO_API` (and also the uses you added above and below).